tests/vmi_configuration_test.go

```diff
@@ -269,22 +269,6 @@ var _ = Describe("Configurations", func() {
 				Expect(domXml).To(ContainSubstring("driver name='vhost' queues='3'"))
 			})
 
-			It("[test_id:1666]should reject virtio block queues without cores", func() {
```

Comment:
Now we only need an e2e test which ensures that we really set 100m by default. A simple test which starts a VMI and where we simply check if `100m` got set should be good enough.

Reply:
Done